packages/@react-aria/interactions/src/utils.ts

```diff
@@ -30,6 +30,13 @@ export function isVirtualClick(event: MouseEvent | PointerEvent): boolean {
     return true;
   }
 
+  // Android TalkBack's detail value varies depending on the event listener providing the event so we have specific logic here instead
+  // If pointerType is defined, event is from a click listener. For events from mousedown listener, detail === 0 is a sufficient check
+  // to detect TalkBack virtual clicks.
+  if (isAndroid() && (event as PointerEvent).pointerType != null) {
```

Comment:
But how is this ever true? click events don't have a `pointerType` property...

Reply:
Also in what scenario was the previous one broken? I just tried double tapping a button with talkback and the onPress event reported virtual.

Reply:
If it is React's synthetic click event, then yes it won't have a `pointerType`, but if we provide the native event or use a click listener via `addEventListener` then click events have a pointerType property. The previous scenario that was broken was if the user double tapped the drag handle in the first list of [this story](https://reactspectrum.blob.core.windows.net/reactspectrum/96f765a9161edb095d84406a6c27d38d538ae04f/storybook/index.html?path=/story/listview-drag-and-drop-util-handlers--complex-drag-between-lists&providerSwitcher-toastPosition=bottom) with TalkBack. Note that swiping left and right will move focus onto the row insertion indicators in the first list even though this list doesn't accept reorder operations. This is because `handleClickEvent` in `useFocusVisible` doesn't set modality to "virtual" since `isVirtualClick` returns false without the current fix, meaning `beginDragging` in `DragManager` doesn't call `setCurrentDropTarget` and thus doesn't call useDroppableCollection's `onDropEnter` to set the `setDropCollectionRef` breaking the default getDropOperation's logic for detecting valid internal drop targets

Reply:
That seems like a Chrome bug. The click event should fire a MouseEvent, not a PointerEvent. In every other browser, pointerType is undefined: https://codepen.io/devongovett/pen/ZEoKNgr

Reply:
Ah yep, I see that now. I can dig to see if there is a pre-existing bug for this and file one if it doesn't exist, but for now we'll need to keep this logic

Reply:
My only question is why this doesn't happen with usePress? That seems to emit virtual properly...

Reply:
We check for virtual clicks internally for usePress, `isVirtualClick` is only used as a fallback: 
onPointerDown's logic sets `state.pointerType = 'virtual'` https://github.com/adobe/react-spectrum/blob/e514827f49696a9c20fb819bd9610651f7017039/packages/%40react-aria/interactions/src/usePress.ts#L322-L325  -> click handler logic https://github.com/adobe/react-spectrum/blob/e514827f49696a9c20fb819bd9610651f7017039/packages/%40react-aria/interactions/src/usePress.ts#L274-L282